meta/internal/meta.proto

```diff
@@ -0,0 +1,237 @@
+}
+
+message DeleteNodeCommand {
+    extend Command {
+        required DeleteNodeCommand command = 103;
+    }
+	required string ID = 1;
+}
+
+message CreateDatabaseCommand {
+    extend Command {
+        required CreateDatabaseCommand command = 104;
+    }
+	required string Name = 1;
+}
+
+message DropDatabaseCommand {
+    extend Command {
+        required DropDatabaseCommand command = 105;
+    }
+	required string Name = 1;
+}
+
+message CreateDatabaseIfNotExistsCommand {
+    extend Command {
+        required CreateDatabaseIfNotExistsCommand command = 106;
+    }
+	required string Name = 1;
+}
+
+message CreateRetentionPolicyCommand {
+    extend Command {
+        required CreateRetentionPolicyCommand command = 107;
+    }
+	required string Database = 1;
+	required RetentionPolicyInfo RetentionPolicy = 2;
+}
+
+message CreateRetentionPolicyIfNotExistsCommand {
+    extend Command {
+        required CreateRetentionPolicyIfNotExistsCommand command = 108;
+    }
+	required string Database = 1;
+	required RetentionPolicyInfo RetentionPolicy = 2;
+}
+
+message DeleteRetentionPolicyCommand {
+    extend Command {
+        required DeleteRetentionPolicyCommand command = 109;
+    }
+	required string Database = 1;
+	required string Name = 2;
+}
+
+message SetDefaultRetentionPolicyCommand {
+    extend Command {
+        required SetDefaultRetentionPolicyCommand command = 110;
+    }
+	required string Database = 1;
+	required string Name = 2;
+}
+
+message UpdateRetentionPolicyCommand {
+    extend Command {
+        required UpdateRetentionPolicyCommand command = 111;
+    }
+	required string Database = 1;
+	required string Name = 2;
+	optional string NewName = 3;
+	optional int64 Duration = 4;
+	optional uint32 ReplicaN = 5;
+}
+
+message CreateShardGroupIfNotExistsCommand {
+    extend Command {
+        required CreateShardGroupIfNotExistsCommand command = 112;
+    }
+    required string Database = 1;
+    required string Policy = 2;
+    required int64 Timestamp = 3;
+}
+
+message CreateUserCommand {
+    extend Command {
+        required CreateUserCommand command = 113;
+    }
+    required string Username = 1;
+    required string Password = 2;
+    required bool Admin = 3;
+}
+
+message DeleteUserCommand {
+    extend Command {
+        required DeleteUserCommand command = 114;
+    }
+    required string Username = 1;
+}
+
+message UpdateUserCommand {
+    extend Command {
+        required UpdateUserCommand command = 115;
+    }
+    required string Username = 1;
+    required string Password = 2;
+}
+
+message SetPrivilegeCommand {
+    extend Command {
+        required SetPrivilegeCommand command = 116;
+    }
+    required string Username = 1;
+    required UserPrivilege Privilege = 2;
+}
+
+message DeleteShardGroupCommand {
+    extend Command {
+        required DeleteShardGroupCommand command = 117;
+    }
+    required string Database = 1;
+    required string Policy = 2;
+    required uint64 ShardID = 3;
+}
```

Comment:
This weird `extend` format is used so we can do polymorphism with the commands in protobufs.